tests/unit/list/dlist.c

```diff
@@ -167,7 +167,7 @@ static inline bool verify_tail_head(sys_dlist_t *list,
  */
 
 /**
- * @brief Verify doubly linked list funtionalities
+ * @brief Verify doubly linked list functionalities
```

Comment:
Please add new misspellings in a separate patch.

Reply:
OK, I will do this in a separate patch.